packages/matter.js/src/cluster/interface/LevelControlInterface.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*** THIS FILE IS GENERATED, DO NOT EDIT ***/
+
+import { LevelControlCluster, ClusterInterface } from "../index.js";
+import { TypeFromSchema } from "../../tlv/TlvSchema.js";
+
+type MoveToLevelRequest = TypeFromSchema<typeof LevelControlCluster.commands.moveToLevel.requestSchema>;
+type MoveRequest = TypeFromSchema<typeof LevelControlCluster.commands.move.requestSchema>;
+type StepRequest = TypeFromSchema<typeof LevelControlCluster.commands.step.requestSchema>;
+type StopRequest = TypeFromSchema<typeof LevelControlCluster.commands.stop.requestSchema>;
+type MoveToLevelWithOnOffRequest = TypeFromSchema<typeof LevelControlCluster.commands.moveToLevelWithOnOff.requestSchema>;
+type MoveWithOnOffRequest = TypeFromSchema<typeof LevelControlCluster.commands.moveWithOnOff.requestSchema>;
+type StepWithOnOffRequest = TypeFromSchema<typeof LevelControlCluster.commands.stepWithOnOff.requestSchema>;
+type StopWithOnOffRequest = TypeFromSchema<typeof LevelControlCluster.commands.stopWithOnOff.requestSchema>;
+
+export interface Common {
+    readonly currentLevel: number | undefined;
+    readonly minLevel?: number;
+    readonly maxLevel?: number;
+    readonly onOffTransitionTime?: number;
+    readonly onLevel: number | undefined;
+    readonly onTransitionTime?: number | undefined;
+    readonly offTransitionTime?: number | undefined;
+    readonly defaultMoveRate?: number | undefined;
+    readonly options: number;
+    readonly remainingTime?: number;
+    readonly startUpCurrentLevel?: number | undefined;
+
+    invokeMoveToLevel(request: MoveToLevelRequest): Promise<void>;
```

Comment:
The "invoke" should only fo into a "Client" object and correstponding "commandhandling callbacks" into Server

Reply:
Resolved on discord, now generated as "onCommand" for server and "sendCommand" for client.  Went back to "send" for client, less matterish but saves two characters.